src/interfaces/ws.js

```diff
@@ -488,7 +489,7 @@ function processUpdates(app, pathSources, spark, msg) {
     )
     return
   }
-  app.handleMessage(spark.request.source || 'ws', msg)
+  app.handleMessage(clientSourceId(spark), msg)
```

Comment:
shouldn't this be a string? clientSourceId returns an object?

Reply:
Oops.